apps.yml

```diff
@@ -3712,12 +3712,14 @@ apps:
     url: https://stats.voice.mozit.cloud/
 - application:
     authorized_groups:
-    - team_moco
-    - team_mofo
+    - stripe_subplat_admin
+    - stripe_subplat_analyst
+    - stripe_subplat_developer
+    - stripe_subplat_supportsp
```

Comment:
stripe_subplat_viewonly was added in https://bugzilla.mozilla.org/show_bug.cgi?id=1637117#c33 so I think you want that here also?

Reply:
@gcoxmoz Thanks for catching that! 💯